ui/admin/tests/integration/components/worker-diagram/dual-filter/index-test.js

```diff
@@ -5,41 +5,153 @@ import { hbs } from 'ember-cli-htmlbars';
 import { setupIntl } from 'ember-intl/test-support';
 
 module(
-  'Integration | Component | worker-diagram/dual-filter',
+  'Integration | Component | worker-diagram/dual-filter Enterprise',
   function (hooks) {
     setupRenderingTest(hooks);
     setupIntl(hooks);
 
+    let featuresService;
+    const targetWorkerFilterIngress = 'target-worker-filters-v2-ingress';
+    const targetWorkerFilterHCP = 'target-worker-filters-v2-hcp';
+
+    hooks.beforeEach(function () {
+      featuresService = this.owner.lookup('service:features');
+      featuresService.enable(targetWorkerFilterIngress);
+      featuresService.disable(targetWorkerFilterHCP);
+    });
+
     test('it renders the correct diagram when egressFilter is false and ingress is false', async function (assert) {
-      assert.expect(4);
+      assert.expect(6);
       await render(
         hbs`<WorkerDiagram::DualFilter @egressFilter={{false}} @ingressFilter={{false}} />`
       );
 
+      assert
+        .dom('.ordered-series-diagram-item:nth-child(2)')
+        .hasText('Front line worker');
+      assert
+        .dom('.ordered-series-diagram-item:nth-child(2)')
+        .doesNotIncludeText('HCP worker');
```

Comment:
Added `.hasText()` and `.doesNotIncludeText()` asserts due to the translations being controlled by the feature flags. Happy to discuss alternatives if needed.

Reply:
Let's use data test attributes for this?  We should avoid testing on hard coded strings in this manner.  If you really must, lookup the intl service in the test and fetch the value of the string via it's translation key.